content/docs/guides/neon-private-networking.md

```diff
@@ -81,30 +122,142 @@ To configure Neon Private Networking, perform the following steps:
+    <Tabs labels={["Neon Console", "CLI", "API"]}>
 
-    Enter **0.0.0.0** in the allowlist to block all connections over the public internet, and click **Save changes**.
+    <TabItem>
 
-    <Admonition type="note">
-     The Private Networking connection is not affected by this IP Allow configuration.
-    </Admonition>
+    To block access via the Neon Console, you can use Neon's [IP Allow](/docs/introduction/ip-allow) feature.
+
+    You can access your **IP Allow** configuration for your Neon project on the ****Network security** page in your project's **Settings**.
+
+    Enter **0.0.0.0** in the ***Allowed IP addresses and ranges***** to block all connections over the public internet, and click **Save changes**.
 
     ![Neon IP Allow configuration](/docs/guides/pl_neon_ip_allow.png)
 
-    <Admonition type="note">
-     Using the IP allowlist feature for blocking access from the public internet is only for the Private Preview. In the final version of this feature, there will be a dedicated option in the Neon Console for this purpose.
-    </Admonition>
+    </TabItem>
+
+    <TabItem>
+
+    To block access via the Neon CLI, use the [neon projects update](/docs/reference/cli-projects#update) command with the `--block-public-connections` option.
+
+    ```bash
+    neon projects update ---block-public-connections
+    ```
+
+    If you've got more than one Neon project, specify the `--project-id` option with your Neon project ID. You can find you Neon project ID under your project's settings in the Neon Console, or by running this Neon CLI command: `neon projects list`
+
+    </TabItem>
+
+    <TabItem>
+
+    To block access via the Neon API, use the [Update project](https://api-docs.neon.tech/reference/updateproject) endpoint with the `block_public_connections` settings object attribute.
+
+    ```bash
+    curl --request PATCH \
+     --url https://console.neon.tech/api/v2/projects/young-sun-41825641 \
+     --header 'accept: application/json' \
+     --header 'authorization: Bearer $NEON_API_KEY' \
+     --header 'content-type: application/json' \
+     --data '
+    {
+    "project": {
+        "settings": {
+        "block_public_connections": true
+        }
+    }
+    }
+    '
+    ```
+
+    </TabItem>
+
+    </Tabs>
 
 </Steps>
 
+## Assigning a VPC endpoint restriction
+
+You can restrict clients from an AWS VPC from connecting to particular projects in a Neon organization using the Neon CLI or API.
+
+<Tabs labels={["CLI", "API"]}>
+
+<TabItem>
+
+You can specify a CLI command similar to the following to restrict project access:
+
+```bash
+neon vpc project restrict vpce-1234567890abcdef0 --project-id orange-credit-12345678
+```
+
+You will need to provide the VPC endpoint ID that you want to restrict and your Neon project ID. You can find you Neon project ID under your project's settings in the Neon Console, or by running this Neon CLI command: `neon projects list`
```

Comment:
NIT. is it worth mentioning:
```
$ neonctl vpc endpoint status vpce-07cdbc41049d7dcb2 --region-id=aws-eu-central-1 --org-id=org-nameless-block-72040075
┌────────────────────────┬───────┬─────────────────────────┬─────────────────────────────┐
│ Vpc Endpoint Id        │ State │ Num Restricted Projects │ Example Restricted Projects │
├────────────────────────┼───────┼─────────────────────────┼─────────────────────────────┤
│ vpce-07cdbc41049d7dcb2 │ new   │ 1                       │ super-base-59043008         │
└────────────────────────┴───────┴─────────────────────────┴─────────────────────────────┘
```

This whole feature seems to be for experienced developers, so it makes sense to provide as much ways to observe what's going on as possible

Reply:
Added!